### ISSUE TITLE
Increased scatter in extracted lightcurves

### ISSUE THREAD

**Issue by Jayshil** [kevin218/Eureka#485]
### Instrument

NIRSpec (Stages 1-3), Light curve fitting (Stages 4-6)

### What happened?

Hi,
I am trying to reproduce ERS paper results with `Eureka!` (WASP-39 transit data with NIRSpec). I am using the latest version of `Eureka!` (7 Nov). The code is running perfectly but I am having trouble with the results: with the scatter (measured with MAD) of lightcurves. I find that although the scatter in white-light lightcurve is somewhat reasonable (MAD ~ 3500 ppm), that of lightcurves in some of the spectral bins can be much high (~20000 ppm, in some cases, see, the attached plot). I am not sure what is going on here (or, is this supposed to happen?). I would really appreciate any help regarding this!

Note that, in general, the scatter of lightcurve in various spectral bins would range from ~5000 ppm to ~20000 ppm.

For your reference, you can find my `Eureka!` ecf files here: [https://github.com/Jayshil/jwst-ers/tree/main/WASP-39_4](https://github.com/Jayshil/jwst-ers/tree/main/WASP-39_4).

Thank you in advance,
Jayshil
![fig4102_ch36_1D_LC](https://user-images.githubusercontent.com/40012574/200419733-0ac3e204-271f-4883-82a2-387f467af9aa.png)

### Error traceback output

N/A

### What operating system are you using?

Mac OS Ventura 13.0 (Apple M1 chip)

### What version of Python are you running?

Python 3.9.7

### What Python packages do you have installed?

alabaster                 0.7.12                     py_0    conda-forge
aom                       3.5.0                hf0c8a7f_0    conda-forge
appnope                   0.1.3              pyhd8ed1ab_0    conda-forge
arviz                     0.13.0             pyhd8ed1ab_0    conda-forge
asdf                      2.13.0             pyhd8ed1ab_0    conda-forge
asdf-astropy              0.2.2                    pypi_0    pypi
asdf-coordinates-schemas  0.1.0                    pypi_0    pypi
asdf-standard             1.0.3              pyhd8ed1ab_0    conda-forge
asdf-transform-schemas    0.3.0              pyhd8ed1ab_0    conda-forge
asdf-wcs-schemas          0.1.1                    pypi_0    pypi
asteval                   0.9.27             pyhd8ed1ab_0    conda-forge
astraeus                  0.3                      pypi_0    pypi
astropy                   5.1.1            py39h7cc1f47_3    conda-forge
astropy-healpix           0.7              py39h7cc1f47_2    conda-forge
astroscrappy              1.1.0            py39h7cc1f47_2    conda-forge
asttokens                 2.1.0              pyhd8ed1ab_0    conda-forge
attrs                     22.1.0             pyh71513ae_1    conda-forge
babel                     2.11.0             pyhd8ed1ab_0    conda-forge
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backports                 1.0                        py_2    conda-forge
backports.functools_lru_cache 1.6.4              pyhd8ed1ab_0    conda-forge
backports.zoneinfo        0.2.1            py39h6e9494a_7    conda-forge
batman-package            2.4.9            py39h7cc1f47_1    conda-forge
bayesicfitting            3.0.1                    pypi_0    pypi
beautifulsoup4            4.11.1             pyha770c72_0    conda-forge
bleach                    5.0.1              pyhd8ed1ab_0    conda-forge
blosc                     1.21.1               h97e831e_3    conda-forge
bokeh                     2.4.3              pyhd8ed1ab_3    conda-forge
bottleneck                1.3.5            py39h7cc1f47_1    conda-forge
brotli                    1.0.9                hb7f2c08_8    conda-forge
brotli-bin                1.0.9                hb7f2c08_8    conda-forge
brotlipy                  0.7.0           py39ha30fb19_1005    conda-forge
brunsli                   0.1                  h046ec9c_0    conda-forge
bzip2                     1.0.8                h0d85af4_4    conda-forge
c-ares                    1.18.1               h0d85af4_0    conda-forge
c-blosc2                  2.4.3                had5f71b_0    conda-forge
ca-certificates           2022.9.24            h033912b_0    conda-forge
cached-property           1.5.2                hd8ed1ab_1    conda-forge
cached_property           1.5.2              pyha770c72_1    conda-forge
ccdproc                   2.3.1              pyhd8ed1ab_0    conda-forge
celerite                  0.4.2            py39hecff1ad_3    conda-forge
certifi                   2022.9.24          pyhd8ed1ab_0    conda-forge
cffi                      1.15.1           py39h131948b_2    conda-forge
cfitsio                   4.1.0                h2c97ad1_0    conda-forge
cftime                    1.6.2            py39h7cc1f47_1    conda-forge
charls                    2.3.4                he49afe7_0    conda-forge
charset-normalizer        2.1.1              pyhd8ed1ab_0    conda-forge
click                     8.1.3           unix_pyhd8ed1ab_2    conda-forge
cloudpickle               2.2.0              pyhd8ed1ab_0    conda-forge
colorama                  0.4.6              pyhd8ed1ab_0    conda-forge
contourpy                 1.0.6            py39h92daf61_0    conda-forge
corner                    2.2.1              pyhd8ed1ab_0    conda-forge
coverage                  6.5.0            py39ha30fb19_1    conda-forge
crds                      11.16.16                 pypi_0    pypi
cryptography              38.0.3           py39h7eb6a14_0    conda-forge
curl                      7.86.0               h57eb407_1    conda-forge
cycler                    0.11.0             pyhd8ed1ab_0    conda-forge
cython                    0.29.32          py39h7a8716b_1    conda-forge
cytoolz                   0.12.0           py39ha30fb19_1    conda-forge
dask-core                 2022.10.2          pyhd8ed1ab_0    conda-forge
dav1d                     1.0.0                hac89ed1_1    conda-forge
decorator                 5.1.1              pyhd8ed1ab_0    conda-forge
defusedxml                0.7.1              pyhd8ed1ab_0    conda-forge
docutils                  0.17.1                   pypi_0    pypi
drizzle                   1.13.6                   pypi_0    pypi
dynesty                   2.0.1              pyhd8ed1ab_0    conda-forge
emcee                     3.1.3              pyhd8ed1ab_0    conda-forge
entrypoints               0.4                pyhd8ed1ab_0    conda-forge
eureka                    0.7.dev12+gb8361759.d20221107          pypi_0    pypi
exceptiongroup            1.0.1              pyhd8ed1ab_0    conda-forge
executing                 1.2.0              pyhd8ed1ab_0    conda-forge
exotic-ld                 2.0.0                    pypi_0    pypi
filelock                  3.8.0                    pypi_0    pypi
fonttools                 4.38.0           py39ha30fb19_1    conda-forge
freetype                  2.12.1               h3f81eb7_0    conda-forge
fsspec                    2022.10.0          pyhd8ed1ab_0    conda-forge
future                    0.18.2             pyhd8ed1ab_6    conda-forge
george                    0.4.0            py39hf72b562_2    conda-forge
giflib                    5.2.1                hbcb3906_2    conda-forge
gwcs                      0.18.2                   pypi_0    pypi
h5netcdf                  1.0.2                    pypi_0    pypi
h5py                      3.1.0           nompi_py39hdc2b67d_100    conda-forge
hdf4                      4.2.15               h7aa5921_5    conda-forge
hdf5                      1.10.6          nompi_hc5d9132_1114    conda-forge
hypothesis                6.56.4             pyha770c72_0    conda-forge
idna                      3.4                pyhd8ed1ab_0    conda-forge
image-registration        0.2.7.dev15+g05ece9f          pypi_0    pypi
imagecodecs               2022.9.26        py39h1e0cc1c_3    conda-forge
imageio                   2.22.0             pyhfa7a67d_0    conda-forge
imagesize                 1.4.1              pyhd8ed1ab_0    conda-forge
importlib-metadata        5.0.0              pyha770c72_1    conda-forge
importlib-resources       5.10.0             pyhd8ed1ab_0    conda-forge
importlib_resources       5.10.0             pyhd8ed1ab_0    conda-forge
iniconfig                 1.1.1              pyh9f0ad1d_0    conda-forge
ipython                   8.6.0              pyhd1c38e8_1    conda-forge
jedi                      0.18.1             pyhd8ed1ab_2    conda-forge
jinja2                    3.1.2              pyhd8ed1ab_1    conda-forge
jmespath                  1.0.1              pyhd8ed1ab_0    conda-forge
joblib                    1.2.0              pyhd8ed1ab_0    conda-forge
jpeg                      9e                   hac89ed1_2    conda-forge
jsonschema                4.9.1              pyhd8ed1ab_0    conda-forge
jupyter_client            7.4.4              pyhd8ed1ab_0    conda-forge
jupyter_core              4.11.2           py39h6e9494a_0    conda-forge
jupyterlab_pygments       0.2.2              pyhd8ed1ab_0    conda-forge
jwst                      1.6.0                    pypi_0    pypi
jxrlib                    1.1                  h35c211d_2    conda-forge
kiwisolver                1.4.4            py39h92daf61_1    conda-forge
krb5                      1.19.3               hb49756b_0    conda-forge
lcms2                     2.14                 h90f4b2a_0    conda-forge
lerc                      4.0.0                hb486fe8_0    conda-forge
libaec                    1.0.6                he49afe7_0    conda-forge
libavif                   0.11.1               he9a59b1_0    conda-forge
libblas                   3.9.0           16_osx64_openblas    conda-forge
libbrotlicommon           1.0.9                hb7f2c08_8    conda-forge
libbrotlidec              1.0.9                hb7f2c08_8    conda-forge
libbrotlienc              1.0.9                hb7f2c08_8    conda-forge
libcblas                  3.9.0           16_osx64_openblas    conda-forge
libcurl                   7.86.0               h57eb407_1    conda-forge
libcxx                    14.0.6               hccf4f1f_0    conda-forge
libdeflate                1.14                 hb7f2c08_0    conda-forge
libedit                   3.1.20191231         h0678c8f_2    conda-forge
libev                     4.33                 haf1e3a3_1    conda-forge
libffi                    3.4.2                h0d85af4_5    conda-forge
libgfortran               5.0.0           10_4_0_h97931a8_25    conda-forge
libgfortran5              11.3.0              h082f757_25    conda-forge
liblapack                 3.9.0           16_osx64_openblas    conda-forge
libnetcdf                 4.8.1           nompi_hb4d10b0_100    conda-forge
libnghttp2                1.47.0               h7cbc4dc_1    conda-forge
libopenblas               0.3.21          openmp_h429af6e_3    conda-forge
libpng                    1.6.38               ha978bb4_0    conda-forge
libsodium                 1.0.18               hbcb3906_1    conda-forge
libsqlite                 3.39.4               ha978bb4_0    conda-forge
libssh2                   1.10.0               h7535e13_3    conda-forge
libtiff                   4.4.0                hdb44e8a_4    conda-forge
libwebp-base              1.2.4                h775f41a_0    conda-forge
libxcb                    1.13              h0d85af4_1004    conda-forge
libzip                    1.9.2                h3ad4413_1    conda-forge
libzlib                   1.2.13               hfd90126_4    conda-forge
libzopfli                 1.0.3                h046ec9c_0    conda-forge
llvm-openmp               14.0.4               ha654fa7_0    conda-forge
lmfit                     1.0.3              pyhd8ed1ab_0    conda-forge
locket                    1.0.0              pyhd8ed1ab_0    conda-forge
lxml                      4.9.1                    pypi_0    pypi
lz4-c                     1.9.3                he49afe7_1    conda-forge
markdown-it-py            2.1.0                    pypi_0    pypi
markupsafe                2.1.1            py39ha30fb19_2    conda-forge
matplotlib                3.6.2            py39h6e9494a_0    conda-forge
matplotlib-base           3.6.2            py39hb2f573b_0    conda-forge
matplotlib-inline         0.1.6              pyhd8ed1ab_0    conda-forge
mdit-py-plugins           0.3.1                    pypi_0    pypi
mdurl                     0.1.2                    pypi_0    pypi
mistune                   2.0.4              pyhd8ed1ab_0    conda-forge
munkres                   1.1.4              pyh9f0ad1d_0    conda-forge
myst-parser               0.18.1                   pypi_0    pypi
nbclient                  0.7.0              pyhd8ed1ab_0    conda-forge
nbconvert                 7.2.3              pyhd8ed1ab_0    conda-forge
nbconvert-core            7.2.3              pyhd8ed1ab_0    conda-forge
nbconvert-pandoc          7.2.3              pyhd8ed1ab_0    conda-forge
nbformat                  5.7.0              pyhd8ed1ab_0    conda-forge
nbsphinx                  0.8.9              pyhd8ed1ab_0    conda-forge
ncurses                   6.3                  h96cf925_1    conda-forge
nest-asyncio              1.5.6              pyhd8ed1ab_0    conda-forge
netcdf4                   1.5.7           nompi_py39h8cf34ae_101    conda-forge
networkx                  2.8.8              pyhd8ed1ab_0    conda-forge
numpy                     1.23.4           py39hdfa1d0c_1    conda-forge
numpydoc                  1.5.0                    pypi_0    pypi
opencv-python             4.6.0.66                 pypi_0    pypi
openjpeg                  2.5.0                h5d0d7b0_1    conda-forge
openssl                   1.1.1s               hfd90126_0    conda-forge
packaging                 21.3               pyhd8ed1ab_0    conda-forge
pandas                    1.5.1            py39hecff1ad_1    conda-forge
pandoc                    2.19.2               h694c41f_1    conda-forge
pandocfilters             1.5.0              pyhd8ed1ab_0    conda-forge
parsley                   1.3                      pypi_0    pypi
parso                     0.8.3              pyhd8ed1ab_0    conda-forge
partd                     1.3.0              pyhd8ed1ab_0    conda-forge
pexpect                   4.8.0              pyh1a96a4e_2    conda-forge
photutils                 1.5.0            py39h15b18c7_0    conda-forge
pickleshare               0.7.5                   py_1003    conda-forge
pillow                    9.2.0            py39h35d4919_3    conda-forge
pip                       22.3.1             pyhd8ed1ab_0    conda-forge
pkgutil-resolve-name      1.3.10             pyhd8ed1ab_0    conda-forge
pluggy                    1.0.0              pyhd8ed1ab_5    conda-forge
poppy                     1.0.3                    pypi_0    pypi
prompt-toolkit            3.0.32             pyha770c72_0    conda-forge
psutil                    5.9.3            py39ha30fb19_1    conda-forge
pthread-stubs             0.4               hc929b4f_1001    conda-forge
ptyprocess                0.7.0              pyhd3deb0d_0    conda-forge
pure_eval                 0.2.2              pyhd8ed1ab_0    conda-forge
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pyerfa                    2.0.0.1          py39h7cc1f47_3    conda-forge
pygments                  2.13.0             pyhd8ed1ab_0    conda-forge
pyopenssl                 22.1.0             pyhd8ed1ab_0    conda-forge
pyparsing                 3.0.9              pyhd8ed1ab_0    conda-forge
pyrsistent                0.19.2           py39ha30fb19_0    conda-forge
pysocks                   1.7.1              pyha2e5f31_6    conda-forge
pysynphot                 2.0.0                    pypi_0    pypi
pytest                    7.2.0              pyhd8ed1ab_2    conda-forge
pytest-arraydiff          0.3                        py_0    conda-forge
pytest-astropy            0.10.0             pyhd8ed1ab_0    conda-forge
pytest-astropy-header     0.1.2                      py_0    conda-forge
pytest-cov                4.0.0              pyhd8ed1ab_0    conda-forge
pytest-doctestplus        0.12.1             pyhd8ed1ab_0    conda-forge
pytest-filter-subpackage  0.1.1                      py_0    conda-forge
pytest-mock               3.10.0             pyhd8ed1ab_0    conda-forge
pytest-openfiles          0.5.0                      py_0    conda-forge
pytest-remotedata         0.3.3              pyhd8ed1ab_0    conda-forge
python                    3.9.7           h1248fe1_3_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python-fastjsonschema     2.16.2             pyhd8ed1ab_0    conda-forge
python_abi                3.9                      2_cp39    conda-forge
pytz                      2022.6             pyhd8ed1ab_0    conda-forge
pywavelets                1.3.0            py39h7cc1f47_2    conda-forge
pyyaml                    6.0              py39ha30fb19_5    conda-forge
pyzmq                     24.0.1           py39hed8f129_1    conda-forge
readline                  8.1.2                h3899abd_0    conda-forge
reproject                 0.9.1            py39h7cc1f47_2    conda-forge
requests                  2.28.1             pyhd8ed1ab_1    conda-forge
scikit-image              0.19.3           py39hecff1ad_2    conda-forge
scikit-learn              1.1.3            py39hdbdcc14_1    conda-forge
scipy                     1.9.3            py39h8a15683_1    conda-forge
semantic_version          2.10.0             pyhd8ed1ab_0    conda-forge
setuptools                65.5.1             pyhd8ed1ab_0    conda-forge
setuptools-scm            7.0.5              pyhd8ed1ab_1    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
snappy                    1.1.9                h225ccf5_2    conda-forge
snowballstemmer           2.2.0              pyhd8ed1ab_0    conda-forge
sortedcontainers          2.4.0              pyhd8ed1ab_0    conda-forge
soupsieve                 2.3.2.post1        pyhd8ed1ab_0    conda-forge
spherical-geometry        1.2.23                   pypi_0    pypi
sphinx                    5.3.0              pyhd8ed1ab_0    conda-forge
sphinx-rtd-theme          1.1.1                    pypi_0    pypi
sphinxcontrib-applehelp   1.0.2                      py_0    conda-forge
sphinxcontrib-devhelp     1.0.2                      py_0    conda-forge
sphinxcontrib-htmlhelp    2.0.0              pyhd8ed1ab_0    conda-forge
sphinxcontrib-jsmath      1.0.1                      py_0    conda-forge
sphinxcontrib-qthelp      1.0.3                      py_0    conda-forge
sphinxcontrib-serializinghtml 1.1.5              pyhd8ed1ab_2    conda-forge
sqlite                    3.39.4               h9ae0607_0    conda-forge
stack_data                0.6.0              pyhd8ed1ab_0    conda-forge
stcal                     1.2.1                    pypi_0    pypi
stdatamodels              0.4.3                    pypi_0    pypi
stpipe                    0.4.2                    pypi_0    pypi
stsci-image               2.3.5                    pypi_0    pypi
stsci-imagestats          1.6.3                    pypi_0    pypi
stsci-stimage             0.2.6                    pypi_0    pypi
svo-filters               0.4.4                    pypi_0    pypi
threadpoolctl             3.1.0              pyh8a188c0_0    conda-forge
tifffile                  2022.10.10         pyhd8ed1ab_0    conda-forge
tinycss2                  1.2.1              pyhd8ed1ab_0    conda-forge
tk                        8.6.12               h5dbffcc_0    conda-forge
toml                      0.10.2             pyhd8ed1ab_0    conda-forge
tomli                     2.0.1              pyhd8ed1ab_0    conda-forge
toolz                     0.12.0             pyhd8ed1ab_0    conda-forge
tornado                   6.2              py39ha30fb19_1    conda-forge
tqdm                      4.64.1             pyhd8ed1ab_0    conda-forge
traitlets                 5.5.0              pyhd8ed1ab_0    conda-forge
tweakwcs                  0.8.0                    pypi_0    pypi
typing-extensions         4.4.0                hd8ed1ab_0    conda-forge
typing_extensions         4.4.0              pyha770c72_0    conda-forge
tzdata                    2022f                h191b570_0    conda-forge
uncertainties             3.1.7              pyhd8ed1ab_0    conda-forge
unicodedata2              15.0.0           py39ha30fb19_0    conda-forge
urllib3                   1.26.11            pyhd8ed1ab_0    conda-forge
wcwidth                   0.2.5              pyh9f0ad1d_2    conda-forge
webencodings              0.5.1                      py_1    conda-forge
wheel                     0.38.2             pyhd8ed1ab_0    conda-forge
wiimatch                  0.3.1                    pypi_0    pypi
xarray                    2022.11.0          pyhd8ed1ab_0    conda-forge
xarray-einstats           0.3.0              pyhd8ed1ab_0    conda-forge
xorg-libxau               1.0.9                h35c211d_0    conda-forge
xorg-libxdmcp             1.1.3                h35c211d_0    conda-forge
xz                        5.2.6                h775f41a_0    conda-forge
yaml                      0.2.5                h0d85af4_2    conda-forge
zeromq                    4.3.4                he49afe7_1    conda-forge
zfp                       1.0.0                ha894c9a_3    conda-forge
zipp                      3.10.0             pyhd8ed1ab_0    conda-forge
zlib                      1.2.13               hfd90126_4    conda-forge
zlib-ng                   2.0.6                hac89ed1_0    conda-forge
zstd                      1.5.2                hfa58983_4    conda-forge

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by kevin218** [comment]
Is this using PRISM or G395H data?

**Comment by Jayshil** [comment]
This is PRISM data (proposal id 1366, published in [arXiv:2208.11692](https://arxiv.org/abs/2208.11692)).

**Comment by kevin218** [comment]
You should go back to Stage 1 and look into the curvature correction and saturation steps. There's a lot of work done at Stage 1 for this dataset.

**Comment by Jayshil** [comment]
Thanks for your input! I tried the saturation step in Stage 1 and it did help in reducing overall scatter. Now, MAD is reduced for most of the lightcurves, but still at ~10k-15k ppm for some of the lightcurves (~20k ppm for worse cases at longer wavelengths). How much scatter (in terms of MAD computed by `Eureka!`) approximately should I expect for this dataset?

However, I am not sure if I understand the curvature correction that you mentioned. Isn't that applied only to G395H data and not PRISM data?

Thanks,
Jayshil

**Comment by Jayshil** [comment]
Hi,
I noticed that I am getting the following warning during ramp fitting at Stage 1:

```
2022-11-09 23:26:29,697 - stpipe.Eureka_RampFitStep - WARNING - /Users/japa6985/opt/anaconda3/envs/eureka_dev/lib/python3.9/site-packages/stdatamodels/validate.py:38: ValidationWarning: While validating meta.exposure.p_readpatt the following error occurred:
'NRS|NRSRAPID' does not match '^((ACQ1|ACQ2|BRIGHT1|BRIGHT2|DEEP2|DEEP8|FAST|FASTGRPAVG|FASTGRPAVG8|FASTGRPAVG16|FASTGRPAVG32|FASTGRPAVG64|FASTR1|FASTR100|FGS|FGS60|FGS8370|FGS840|FGSRAPID|FINEGUIDE|ID|MEDIUM2|MEDIUM8|NIS|NISRAPID|NRS|NRSIRS2|NRSN16R4|NRSN32R8|NRSN8R2|NRSRAPID|NRSIRS2RAPID|NRSRAPIDD1|NRSRAPIDD2|NRSRAPIDD6|NRSSLOW|RAPID|SHALLOW2|SHALLOW4|SLOW|SLOWR1|TRACK|ANY|N/A)\\s*\\|\\s*)+$'

Failed validating 'pattern' in schema:
    OrderedDict([('title', 'Readout pattern'),
                 ('type', 'string'),
                 ('pattern',
                  '^((ACQ1|ACQ2|BRIGHT1|BRIGHT2|DEEP2|DEEP8|FAST|FASTGRPAVG|FASTGRPAVG8|FASTGRPAVG16|FASTGRPAVG32|FASTGRPAVG64|FASTR1|FASTR100|FGS|FGS60|FGS8370|FGS840|FGSRAPID|FINEGUIDE|ID|MEDIUM2|MEDIUM8|NIS|NISRAPID|NRS|NRSIRS2|NRSN16R4|NRSN32R8|NRSN8R2|NRSRAPID|NRSIRS2RAPID|NRSRAPIDD1|NRSRAPIDD2|NRSRAPIDD6|NRSSLOW|RAPID|SHALLOW2|SHALLOW4|SLOW|SLOWR1|TRACK|ANY|N/A)\\s*\\|\\s*)+$'),
                 ('fits_keyword', 'P_READPA')])

On instance:
    'NRS|NRSRAPID'
  warnings.warn(errmsg, ValidationWarning)

2022-11-09 23:26:29,742 - stpipe.Eureka_RampFitStep - INFO - Extracting gain subarray to match science data
2022-11-09 23:26:29,743 - stpipe.Eureka_RampFitStep - INFO - Extracting readnoise subarray to match science data
2022-11-09 23:26:29,743 - stpipe.Eureka_RampFitStep - INFO - Using algorithm = default
2022-11-09 23:26:29,743 - stpipe.Eureka_RampFitStep - INFO - Using weighting = default
2022-11-09 23:26:29,794 - stpipe.Eureka_RampFitStep - INFO - Number of processors used for multiprocessing: 8
```

Could this give a high scatter in the lightcurve? Note that the Stage 1 keeps continue despite this warning, but I am not sure if this warning is of any concern.

Thank you,
Jayshil

**Comment by taylorbell57** [comment]
Hi @Jayshil. A little while ago I tried to reproduce the PRISM ERS results myself and was successful in my attempts. Here are ECFs and an EPF that pretty closely match the published ERS PRISM methods and results (with an offset of 79 ppm and some differences in the saturated region which is known to not be very reliable/reproducible). These should work with version 0.9 of Eureka!.

![SpectraComparison](https://github.com/kevin218/Eureka/assets/24514122/5fa2d56a-b245-443e-909c-86335a0565e2)
[ShareECFs.zip](https://github.com/kevin218/Eureka/files/11764108/ShareECFs.zip)

As it seems that there is nothing wrong with the pipeline at this time, I am going to close this issue. For your reference, we intend to publicly host ECFs that are able reasonably reproduce each of the ERS results like the files I've attached above - it's just going to take us some time.
